py/mpconfig.h

```diff
@@ -1779,6 +1779,12 @@ typedef double mp_float_t;
 #define MICROPY_PY_MACHINE_SOFTSPI (0)
 #endif
 
+// if a port didn't define MSB/LSB constants then provide them
```

Comment:
I suggest "Values of SPI.MSB and SPI.LSB constants" (no need to say anything about a port providing them or not, because this whole file is about defining defaults when a port doesn't specify config options).

Reply:
Changed.